Add Support for Angular 11.1+

**Description**
The ng add @ngx-formly/schematics command no longer works. It now throws the error:
`An unhandled exception occurred: Cannot find module '@schematics/angular/utility/config'`

**Minimal Reproduction**

1. Use Angular 11.1 or higher. 
2. Create a new project
3. Run ng add @ngx-formly/schematics

- Angular version: 11.1.4
- Formly version: 5.10.13

**Additional context**
As of Angular CLI 11.1 they have removed schematics/angular/utility/config.

--- a/src/schematics/utils/ast.ts
+++ b/src/schematics/utils/ast.ts
@@ -1,13 +1,29 @@
-import { normalize } from '@angular-devkit/core';
+import { JsonParseMode, normalize, parseJson } from '@angular-devkit/core';
 import { SchematicsException, Tree } from '@angular-devkit/schematics';
 import * as ts from 'typescript';
 import { addImportToModule } from '@schematics/angular/utility/ast-utils';
 import { InsertChange } from '@schematics/angular/utility/change';
-import { getWorkspace } from '@schematics/angular/utility/config';
 import { getAppModulePath } from '@schematics/angular/utility/ng-ast-utils';
 import { findModuleFromOptions as internalFindModule } from '@schematics/angular/utility/find-module';
-import { WorkspaceProject } from '@angular-devkit/core/src/workspace';
+import { WorkspaceProject, WorkspaceSchema } from '@angular-devkit/core/src/workspace';
 
+export function getWorkspacePath(host: Tree): string {
+  const possibleFiles = ['/angular.json', '/.angular.json'];
+  const path = possibleFiles.filter(path => host.exists(path))[0];
+
+  return path;
+}
+
+export function getWorkspace(host: Tree): WorkspaceSchema {
+  const path = getWorkspacePath(host);
+  const configBuffer = host.read(path);
+  if (configBuffer === null) {
+    throw new SchematicsException(`Could not find (${path})`);
+  }
+  const content = configBuffer.toString();
+
+  return parseJson(content, JsonParseMode.Loose) as {} as WorkspaceSchema;
+}
 
 /** Reads file given path and returns TypeScript source file. */
 export function getSourceFile(host: Tree, path: string): ts.SourceFile {

